binder/plugins/models/html_field.py

```diff
@@ -0,0 +1,80 @@
+from django.db.models import TextField
+from html.parser import HTMLParser
+from django.core import exceptions
+
+
+def link_validator(tag, attribute_name, attribute_value):
+	if not attribute_value.startswith('http://') and not attribute_value.startswith('https://'):
+		raise exceptions.ValidationError(
+			'Link is not valid',
+			code='invalid_tag',
+			params={
+				'tag': tag,
+			},
+		)
+
+
+class HtmlValidator(HTMLParser):
+	allowed_tags = [
+		# General setup
+		'p', 'br',
+		# Headers
+		'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'h7',
+
+		# text decoration
+		'b', 'strong', 'i', 'em', 'u',
+		# Lists
+		'ol', 'ul', 'li',
+
+		# Special
+		'a',
+	]
+
+	allowed_attributes = {
+		'a': ['href', 'rel', 'target']
+	}
+
+	special_validators = {
+		('a', 'href'): link_validator
+	}
+
+	error_messages = {
+		'invalid_tag': 'Tag %(tag)s is not allowed',
```

Comment:
Do we want to start using `gettext` to enable translations in binder? we do this in some places and not in others.